common/src/main/java/com/wynntils/handlers/actionbar/ActionBarHandler.java

```diff
@@ -56,32 +52,32 @@ public void onActionBarUpdate(ChatPacketReceivedEvent.GameInfo event) {
         }
         previousRawContent = content;
 
-        Matcher matcher = content.getMatcher(ACTIONBAR_PATTERN);
-        if (!matcher.matches()) {
-            WynntilsMod.warn("ActionBarHandler pattern failed to match: " + content);
-            return;
-        }
+        StyledText[] contentGroups = content.split(" {4,}");
 
         // Create map of position -> matching part of the content
         Map<ActionBarPosition, StyledText> positionMatches = new EnumMap<>(ActionBarPosition.class);
         Arrays.stream(ActionBarPosition.values())
-                .forEach(pos -> positionMatches.put(pos, StyledText.fromString(matcher.group(pos.name()))));
+                .forEach(pos -> positionMatches.put(pos, contentGroups[pos.ordinal()]));
```

Comment:
This assumes the split always results in 3 parts. Is this always true?

Reply:
I cannot think of any actionbar where it is not true. Normally, we always have health + mana, then coordinates, "Housing Island", spring, or powder there. I will add a failsafe though